tests/acceptance/features/apiNotification/notification.feature

```diff
@@ -10,241 +10,420 @@ Feature: Notification
       | Alice    |
       | Brian    |
       | Carol    |
-    And user "Alice" has uploaded file with content "other data" to "/textfile1.txt"
-    And user "Alice" has created folder "my_data"
+    And the administrator has given "Alice" the role "Space Admin" using the settings api
+    And user "Alice" has created a space "notification checking" with the default quota using the GraphApi
 
 
-  Scenario Outline: user gets a notification of resource sharing
-    Given user "Alice" has shared entry "<resource>" with user "Brian"
+  Scenario: get a notification of space shared
+    Given user "Alice" has shared a space "notification checking" with settings:
+      | shareWith | Brian  |
+      | role      | editor |
```

Comment:
Shouldn't this file be for notifications related to resources?

Reply:
i messed up while git push 🥲